src/Microsoft.ML.Data/Transforms/ExplainabilityCatalog.cs

```diff
@@ -18,18 +18,18 @@ public static class ExplainabilityCatalog
         /// </summary>
         /// <param name="catalog">The model explainability operations catalog.</param>
         /// <param name="modelParameters">Trained model parameters that support Feature Contribution Calculation and which will be used for scoring.</param>
-        /// <param name="featureColumn">The name of the feature column that will be used as input.</param>
+        /// <param name="featureColumnName">The name of the feature column that will be used as input.</param>
         /// <param name="numPositiveContributions">The number of positive contributions to report, sorted from highest magnitude to lowest magnitude.
         /// Note that if there are fewer features with positive contributions than <paramref name="numPositiveContributions"/>, the rest will be returned as zeros.</param>
         /// <param name="numNegativeContributions">The number of negative contributions to report, sorted from highest magnitude to lowest magnitude.
         /// Note that if there are fewer features with negative contributions than <paramref name="numNegativeContributions"/>, the rest will be returned as zeros.</param>
         /// <param name="normalize">Whether the feature contributions should be normalized to the [-1, 1] interval.</param>
         public static FeatureContributionCalculatingEstimator FeatureContributionCalculation(this ModelOperationsCatalog.ExplainabilityTransforms catalog,
             ICalculateFeatureContribution modelParameters,
-            string featureColumn = DefaultColumnNames.Features,
```

Comment:
>featureColumn =  [](start = 19, length = 16)

#2966 rewrites these functions, so you can drop changes to `ExplainabilityCatalog`.

Reply:
That would be much appreciated indeed :)